specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/managedHsm.json

```diff
@@ -0,0 +1,634 @@
+        },
+        "createMode": {
+          "type": "string",
+          "description": "The create mode to indicate whether the resource is being created or is being recovered from a deleted resource.",
+          "enum": [
+            "recover",
+            "default"
+          ],
+          "x-ms-enum": {
+            "name": "CreateMode",
+            "modelAsString": false,
+            "values": [
+              {
+                "value": "recover",
+                "description": "Recover the vault from a soft-deleted resource."
+              },
+              {
+                "value": "default",
+                "description": "Create a new vault.  This is the default option."
+              }
+            ]
+          }
+        },
+        "statusMessage": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource Status Message."
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Provisioning state.",
+          "enum": [
+            "Succeeded",
+            "Allocating",
+            "Provisioning",
+            "Failed",
+            "Deleting",
+            "Activated",
+            "SecurityDomainRestore",
+            "Restoring",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Succeeded",
+                "description": "The managed HSM Pool has been full provisioned."
+              },
+              {
+                "value": "Allocating",
+                "description": "Resources are currently being allocated for the managed HSM resource."
+              },
+              {
+                "value": "Provisioning",
+                "description": "The managed HSM Pool is currently being provisioned."
+              },
+              {
+                "value": "Failed",
+                "description": "Provisioning of the managed HSM Pool has failed."
+              },
+              {
+                "value": "Deleting",
+                "description": "The managed HSM Pool is currently being deleted."
+              },
+              {
+                "value": "Activated",
+                "description": "The managed HSM pool is ready for normal use."
+              },
+              {
+                "value": "SecurityDomainRestore",
+                "description": "The managed HSM pool is waiting for a security domain restore action."
+              },
+              {
+                "value": "Restoring",
+                "description": "The managed HSM pool is being restored from full HSM backup."
+              },
+              {
+                "value": "Updating",
```

Comment:
What does this correspond to in HFC / Amit's spec?

Reply:
This is kept for updates made to 'Tags' in RP, but that is a sync call today so removing this state for now.